pyccel/codegen/printing/cucode.py

```diff
@@ -61,14 +61,45 @@ def _print_Module(self, expr):
             else:
                 local_imports += self._print(imp)
 
-        imports = f'{c_headers_imports}\
-                    extern "C"{{\n\
-                    {local_imports}\
-                    }}'
+        imports = (f'{c_headers_imports}\n'
+                   f'{local_imports}\n')
```

Comment:
If we no longer need the `extern "C"` do we still need to separate the imports into `c_header_imports` and `local_imports`? Or could we now simply have:
```python
imports = ''.join(self._print(imp) for imp in imports)
```
?